<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>   Username</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="username">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="user_id">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>    User_Id</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="upload_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>    Upload_images</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="upload_button">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload  images</string>
    </property>
   </widget>
   <widget class="QLabel" name="instruction">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>227</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Should be Upload &gt;5 &amp; &lt;10 human's face images</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_msgs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>521</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(239, 41, 41);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="camera_btn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Take Pictures</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>187</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>or</string>
    </property>
   </widget>
   <widget class="QLabel" name="video">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>266</y>
      <width>511</width>
      <height>391</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="done_btn">
    <property name="geometry">
     <rect>
      <x>478</x>
      <y>190</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
   <widget class="QLabel" name="id">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>110</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>555</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdmin_Panel">
    <property name="font">
     <font>
      <family>Serif</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>Admin Panel</string>
    </property>
   </widget>
   <addaction name="menuAdmin_Panel"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
